general/source-code-conventions.md

```diff
@@ -0,0 +1,80 @@
+# Source code conventions
+Adhere to these general source code conventions unless otherwise specified in language specific guide.
+
+## Files
+- Use language specific indentation.
+- If you have to use tabs, do not mix tabs and spaces in a single project.
+- All files end with an empty line.
```

Comment:
All files na user created kailangan may extra space sa dulo? What's the reason for this and how do we handle IDE/system generated files that are user accessible?